### PR TITLE
Doc: Updated documentation for Ollama

### PR DESCRIPTION
**Pull request by JayQuimby**
## Doc change:
- Simplified guide using Ollama server instead of litellm server
- Updated guide to be in correct folder `docs`
- General readability improvements

### CODE REVIEW

**Comment by rbren** [comment]
This seems like the wrong fix--I think it's broken for more than just local LLM users. I believe we have another PR in the works to fix this issue

**Comment by JayQuimby** [comment]
@rbren changed this so its only documentation update.

File: docs/examples/LOCAL_LLM_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+curl -fsSL https://ollama.com/install.sh | sh
+```
+Windows or macOS: 
+
+- Download from [here](https://ollama.com/download/)
+
+## 1. Install Models:
+Ollama model names can be found [here](https://ollama.com/library) (See example below)
+
+![alt text](images/ollama.png)
```
Comment (rbren) [inline docs/examples/LOCAL_LLM_GUIDE.md:17]: looks like we lost `example.png`. Should we delete it?

Comment (JayQuimby) [reply docs/examples/LOCAL_LLM_GUIDE.md:17]: Yes, it is no longer necessary.

**Comment by rbren** [comment]
Thank you!

**Pull request merged**